roles/openshift_logging/defaults/main.yml

```diff
@@ -99,7 +99,7 @@ openshift_logging_es_log_appenders: ['file']
 openshift_logging_es_memory_limit: "8Gi"
 openshift_logging_es_pv_selector: "{{ openshift_logging_storage_labels | default('') }}"
 openshift_logging_es_pvc_dynamic: "{{ openshift_logging_elasticsearch_pvc_dynamic | default(False) }}"
-openshift_logging_es_pvc_size: "{{ openshift_logging_elasticsearch_pvc_size | default('') }}"
```

Comment:
why are we removing this?

Reply:
that is what we claim in our [readme](https://github.com/openshift/openshift-ansible/blob/6c4c57e1ef15d2153f91c747f8a4718be49dafda/roles/openshift_logging/README.md) currently. Also, we later propagate to the DC as `openshift_logging_elasticsearch_pvc_size: "{{openshift_logging_es_pvc_size}}"`. If we keep this around, it creates code confusing to read and kind of a loop, even though it evaluets serially, clearly, and there is no loop. It simply reads differently from the way it evaluates.
```
defaulting to:
openshift_logging_es_pvc_size: "{{ openshift_logging_elasticsearch_pvc_size }}"

later propagating as:
openshift_logging_elasticsearch_pvc_size: "{{ openshift_logging_es_pvc_size }}"
or:
openshift_logging_elasticsearch_pvc_size: "{{ openshift_logging_es_ops_pvc_size }}"
```

The way I understand these vars is following, please correct me if it's wrong:
```
openshift_logging_es_* : user facing variable setting properties for ES cluster
openshift_logging_es_ops_* : user facing variable setting properties for ES OPS cluster
openshift_logging_elasticsearch_* : internal variable which gets populated from either 'openshift_logging_es_*' or 'openshift_logging_es_ops_*' depending on which cluster it currently deploys
```

Reply:
Doesn't that remove our ability to override the value being passed in using the `openshift_logging_elasticsearch_pvc_size` variable though?

I don't specifically think of `openshift_logging_elasticsearch_*` variables as "internal". Users are free to set any variable they choose. I think if we wanted to make them more internal we should do something like prefix them with something like `r_` or `l_` (role or local) and not have them in `default/main` or any of the `vars/` files. But that should probably be a proposal/discussion first if we go that route.

Reply:
what does it bring to let users set both `_es_` and `_elasticsearch_`? what behavior would you expect? Documented is only `_es_` afaik

Reply:
ack, if only `_es_` is documented its likely alright. I wasn't meaning to seem like I was blocking on this, was simply wondering since we'd be removing a variable that could be used for setting a value.